checkST2110.js

```diff
@@ -971,210 +1132,481 @@ const test_30_62_4 = (sdp, params) => {
         errors.push(new Error(`Line ${x + 1}: For stream ${streamCount}, payload type of 'fmtp' attribute '${fmtpMatch[1]}' does not match that of the stream '${payloadType}'.`));
       }
       let order = fmtpMatch[2];
-      if (params.should === true && !order.startsWith('SMPTE2110')) {
-        errors.push(new Error(`Line ${x + 1}: For stream ${streamCount}, format parameter 'channel-order' should be specified by the 'SMPTE2110' convention, not '${order.split('.')[0]}', as per SMPTE ST 2110-30 Section 6.2.2.`));
+      if (params.should && !order.startsWith('SMPTE2110')) {
+        errors.push(new Error(`Line ${x + 1}: For stream ${streamCount}, format parameter 'channel-order' should be specified by the 'ST 2110' convention, not '${order.split('.')[0]}', as per ST 2110-30 Section 6.2.2.`));
```

Comment:
This is a reserved value not a simple ref to the spec, so we should revert this like so:

```suggestion
        errors.push(new Error(`Line ${x + 1}: For stream ${streamCount}, format parameter 'channel-order' should be specified by the 'SMPTE2110' convention, not '${order.split('.')[0]}', as per ST 2110-30 Section 6.2.2.`));
```